packages/neon-core/src/tx/transaction/Transaction.ts

```diff
@@ -289,8 +286,15 @@ export class Transaction {
   }
 
   public getScriptHashesForVerifying(): string[] {
```

Comment:
whats the use of this again?

Reply:
Get all the scripthashes that need to sign this transaction, it's used in verification. It'll be used in TransactionBuilder.